library/src/scala/runtime/stdLibPatches/language.scala

```diff
@@ -18,6 +18,7 @@ object language:
    *
    *  @group experimental
    */
+  @scala.annotation.experimental
```

Comment:
I am not sure yet that the `experimental` language object should be `@experimental`. We'd have to carefully compare notes what does what and be sure that it's exactly the same thing.

Reply:
The idea here is that we could remove an experimental feature at some point and that would not be a breaking change. We do check that all of these are explicitly anyway. I could remove it.